### PR TITLE
fix(config): add temp and humidity calibration params to Aeotec aërQ ZWA039

### PR DESCRIPTION
**Pull request by robfarmergt**
This PR adds param 14 and 15 to the Aeotec aerQ Temperature and Humidity sensor.

https://help.aeotec.com/support/solutions/articles/6000227918-a%C3%ABrq-temperature-and-humidity-sensor-user-guide-

### CODE REVIEW

File: packages/config/config/devices/0x0371/zwa039.json [hunk 1]
```diff
@@ -233,6 +233,40 @@
 			"maxValue": 10,
 			"defaultValue": 0
 		},
+    {
+			"#": "14",
+			"label": "Offset value for Temperature",
+			"description": "Can add or minus this setting value to calibrate temperature when checked. Allowable range: -200 to 200.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]:
```suggestion
			"label": "Temperature Calibration",
```

File: packages/config/config/devices/0x0371/zwa039.json [hunk 2]
```diff
@@ -233,6 +233,40 @@
...
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "15",
+			"label": "Offset value for Humidity",
+			"description": "Can add or minus this setting value to calibrate humidity when checked.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]:
```suggestion
			"label": "Humidity Calibration",
```

File: packages/config/config/devices/0x0371/zwa039.json [hunk 3]
```diff
@@ -233,6 +233,40 @@
 			"maxValue": 10,
 			"defaultValue": 0
 		},
+    {
+			"#": "14",
+			"label": "Offset value for Temperature",
+			"description": "Can add or minus this setting value to calibrate temperature when checked. Allowable range: -200 to 200.",
+			"unit": "0.1 °(C/F)",
+			"unsigned": false,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]:
```suggestion
```

File: packages/config/config/devices/0x0371/zwa039.json [hunk 4]
```diff
@@ -233,6 +233,40 @@
...
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "15",
+			"label": "Offset value for Humidity",
+			"description": "Can add or minus this setting value to calibrate humidity when checked.",
+			"unit": "%",
+			"unsigned": false,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]:
```suggestion
```

File: packages/config/config/devices/0x0371/zwa039.json [hunk 5]
```diff
@@ -233,6 +233,40 @@
...
+			"valueSize": 2,
+			"minValue": -200,
+			"maxValue": 200,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]: I don't think we need this option here.

File: packages/config/config/devices/0x0371/zwa039.json [hunk 6]
```diff
@@ -233,6 +233,40 @@
...
+			"valueSize": 1,
+			"minValue": -50,
+			"maxValue": 50,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa039.json]: I don't think we need this option here.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Also, the whitespace is off, but we can have the bot fix this once you're done.

**Comment by robfarmergt** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]

**Comment by robfarmergt** [comment]
@AlCalzone Thanks for the assistance an all the hard work on this project!

**Pull request merged**